.cirrus.yml

```diff
@@ -296,7 +296,7 @@ docker_alpine_3_12_task:
   build_script:
     - ninja -j4 -C build install install-spicy-analyzers
   test_install_script:
-    - SPICY_INSTALLATION_DIRECTORY=/opt/spicy make -C tests test-all-install
+    - SPICY_INSTALLATION_DIRECTORY=/opt/spicy make -C tests test-spicy-install
```

Comment:
Similar to above, use `test-install` -- except that that doesn't exist yet, so let's add that as a shortcut for `test-spicy-install`.